ext/aom.cmd

```diff
@@ -14,6 +14,11 @@ cd aom
 mkdir build.libavif
 cd build.libavif
 
-cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DENABLE_DOCS=0 -DENABLE_EXAMPLES=0 -DENABLE_TESTDATA=0 -DENABLE_TESTS=0 -DENABLE_TOOLS=0 ..
+if [[ `uname` == 'Darwin' && `uname -m` == 'arm64' ]]; then
+  cmake -G Ninja -DCONFIG_RUNTIME_CPU_DETECT=0 -DCMAKE_BUILD_TYPE=Release -DENABLE_DOCS=0 -DENABLE_EXAMPLES=0 -DENABLE_TESTDATA=0 -DENABLE_TESTS=0 -DENABLE_TOOLS=0 ..
+else
+  cmake -G Ninja -DCMAKE_BUILD_TYPE=Release -DENABLE_DOCS=0 -DENABLE_EXAMPLES=0 -DENABLE_TESTDATA=0 -DENABLE_TESTS=0 -DENABLE_TOOLS=0 ..
+fi
+
```

Comment:
This would break batch file compatibility, it might be better to mention this in a BUILDING,md file or similar instead or set  `CONFIG_RUNTIME_CPU_DETECT` to false inside the cmakelist instead